build.gradle

```diff
@@ -38,7 +38,7 @@ ext {
     buildToolsVersion = '28.0.3'
 
     // App dependencies
-    supportLibraryVersion = '1.1.0'
+    supportLibraryVersion = '1.0.0'
```

Comment:
remove this change, keep separate PRs for each issue/change

Reply:
Ok I will. But if I revert this change, the recycler items in saving account, loan account and share account do not work. Before pushing my code I asked the community one of them advised to not remove this change.